features-json/webxr.json

```
@@ -350,9 +350,10 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Chrome 66-72 supported a very early version of the spec",
-    "2":"Part of an origin trial",
-    "3":"Part of the spec can be enabled with the [webxr](chrome://flags/#webxr) and [webxr-hit-test](chrome://flags/#webxr-hit-test) flags."
+    "1":"Chrome 66-72 supported a very early version of the spec.",
+    "2":"Part of an origin trial.",
+    "3":"Part of the spec can be enabled with the [webxr](chrome://flags/#webxr) and [webxr-hit-test](chrome://flags/#webxr-hit-test) flags.",
```

These flags have been gone for a while. I know they're not in Chrome 78 and I have no way to check earlier versions.

---

The recommendation is to remove the flags completely. Users shouldn't be touching them. The implementation differences between 66-77 and 79 versions of the spec are enough that developers should build for 79 or later.

---

Replying to all comments here. I also think this should be removed eventually, but I think you're jumping the gun on it.
My team has been developing solutions for WebXR for a while now, and keeping track of what code runs where and what spec it uses is a nightmare. This resource helps us a lot.
It will take some time for us to update our code to the stable spec, and meanwhile we will still be using the old code.
I imagine other developers are facing the same problem, and i don't think keeping the data will have any negative effect on new development. Anyone new to the game will build for 79+.

---

The flags in note 3 are still present all the way up to v78. They are also present up to v80, though that might change.

---

I'm running version 78 and they're not in my installation. I just double checked.

---

These have already been removed on mac.

---

Ah, that makes sense! I'm on Windows. Browserstack shows they were removed in Chrome for Mac v77.
There are other slight inconsistencies with this note as well, maybe this could work:
```suggestion
    "3":"Chrome 66-78 has had intermittent support behind the #webxr and other #webxr-* flags.",
```
I'd say it's good enough for something that will be removed in a few months?

---

Well let's say I'm not going to argue the point any further. 

This is an example of why I'm never thrilled about this information being in compatibility data. As you can see from this, once a feature ships in stable, we're rather sloppy about when and how we remove the flags. That's a reflection of the fact that we don't intend them for production use.